src/main/java/unicash/model/category/UniqueCategoryList.java

```diff
@@ -199,4 +200,34 @@ public static boolean categoriesAreUnique(List<Category> categories) {
         }
         return true;
     }
+
+    /**
+     * Returns all categories as a single, unified string.
+     *
+     * @return all categories as a string
+     */
+    public String joinCategoriesAsString() {
+        String categoriesString = internalList
+                .stream()
+                .map(category -> category.toString())
+                .collect(Collectors.joining(","));
+
+        requireNonNull(categoriesString);
+        return categoriesString;
+    }
+
+    /**
+     * Returns all categories as a single, unified List.
+     *
+     * @return all categories as a List of categories
+     */
+    public List<Category> joinCategoriesAsList() {
+        List<Category> categoryList = internalList
+                .stream()
+                .collect(Collectors.toList());
+
+        requireNonNull(categoryList);
```

Comment:
Believe `collect()` returns at most an empty list, not a `null` list, so might want to check for that instead

Reply:
Fixed